sites/geohub/src/components/UserAccount.svelte

```diff
@@ -15,6 +15,10 @@
   const handleDropdown = () => {
     dropdownActive = !dropdownActive
   }
+
+  const openSettingsPage = () => {
+    window.location.href = '/settings'
```

Comment:
can we use `goto` method provided by sveltekit?

https://kit.svelte.dev/docs/modules#$app-navigation-goto